.github/actions/spelling/allow/formats.txt

```diff
@@ -0,0 +1,70 @@
+ada
+asm
+asn
+assembly
+bash
+basic
+c
+c#
+c++
```

Comment:
I thought the tool should have stripped out the punctuation here, were these needed? (same with `f#` later)

Reply:
The default minimum word size is `3` characters and in 0.0.21 the only characters it considers are `a-z` and `'` (with some aliases for this character).

Reply:
Resolved with https://github.com/canonical/ubuntu-mir/pull/19/commits/d7312d2433518149bbec0295d625b2a67fb767e3

I added this to `.github/actions/spelling/allow/README.md` with https://github.com/canonical/ubuntu-mir/pull/19/commits/e8224fb04f45d8b89e5df77ba860b4b87658809b

Reply:
Thanks for noticing that!